applications/DEMApplication/tests/forces_and_moments_tests_files/ProjectParametersDEM.json

```diff
@@ -28,28 +28,62 @@
             "python_module" : "processes.apply_forces_and_moments_process",
             "kratos_module" : "KratosMultiphysics.DEMApplication",
             "Parameters"    : {
-                "model_part_name" : "SpheresPart.Ball",
+                "model_part_name" : "SpheresPart.Ball1",
 		"force_settings" : {
-                    "value"                : [100.0, "30.0*t", "2000.0*(x+y)"]
+                    "value"                : [100.0, "30.0*t", "2000.0*(x+y)"],
+                    "table"                : [0, 0, 0]
                 },
                 "moment_settings" : {
-                    "value"                : [-50.0, "15.0*(x+z)", "600.0*t"]
+                    "value"                : [-50.0, "15.0*(x+z)", "600.0*t"],
+                    "table"                : [0, 0, 0]
                 },
                 "interval"             : [0.0, 0.2]
             }
+        },{
+            "python_module" : "processes.apply_forces_and_moments_process",
+            "kratos_module" : "KratosMultiphysics.DEMApplication",
+            "Parameters"    : {
+                "model_part_name" : "SpheresPart.Ball2",
+		"force_settings" : {
+                    "value"                : [0.0, 0.0, 0.0],
+                    "table"                : [1, 2, 3]
```

Comment:
How can we make it evident that we want to use the table and not the value? I am thinking of a key like

`"source" : ["value", "table", "table"]`

Or obliged 'null' in the fields which must not be used by Kratos:
```
"value" : [0.0, null, null],
"table" : [null, 2, 3]
```
I like the second option more.

Reply:
In this process, if value is equal to 0 it does not affect the calculations. For the tables, if the table is 0 is the same as using null in the example you post. Moreover, if you assign a value and a table, the table prevails.
I like what you propose, the second option, I will change it.

Reply:
I have been checking other processes in Kratos and they not use null values for tables, they use int values and skip tables if it is zero.

Reply:
Ok, but here we don't just have an array for the tables, we have one for 'values' and one for 'tables' (two arrays), which can make it confusing, in my opinion. A 0.0 in 'values' can mean that you really want to set the value to 0.0. So we need to put something different from 0.0 at least in the values. If you choose to use '0' instead of 'null' in the tables, it's fine, but I proposed 'null' to use the same trick as in 'values'.
If someone new to Kratos sees
```
"value" : [0.0, 0.0, 0.0],
"table" : [0, 0, 0]
```
he will have many doubts: "is the 0 allowed for tables? Am I fixing to a table or to a value here?" Because he does not know the criterion. If we use a different keyword (like 'null') it is much clearer. Maybe we could ask opinions here.
@joaquinirazabal can you put a link to the example where they skip the tables if the index is 0?

Reply:
By the way, this example is also confusing if you don't know the criterion used:
```
  "value"                : [0.0, 0.0, 0.0],
  "table"                : [1, 2, 3]
```
What are we imposing here? You can only know it if someone tells you the convention, as both the value and the table have possible values.

Reply:
There are not many processes dealing with tables, you can find an example here: https://github.com/KratosMultiphysics/Kratos/blob/master/applications/DamApplication/custom_processes/dam_fix_temperature_condition_process.hpp
I have been thinking a bit more about this and it is not difficult to change it to allow the value null, as you said 'null' is clearer.
About the values, I was thinking only in applying external forces, and in that case, apply zero external force is the same as not applying anything. However, for kinematic constraints apply 0.0 velocity is not the same as not apply anything.

Reply:
Now we can use null in values and tables, it is less confusing.